packages/svelte/src/animate/index.js

```diff
@@ -16,13 +16,21 @@ export function flip(node, { from, to }, params = {}) {
 
 	var transform = style.transform === 'none' ? '' : style.transform;
 	var [ox, oy] = style.transformOrigin.split(' ').map(parseFloat);
+
+	var m = new DOMMatrix(transform);
```

Comment:
It seems like the variable is never mutated, so what about using [DOMMatrixReadOnly](https://developer.mozilla.org/en-US/docs/Web/API/DOMMatrixReadOnly) instead?

```suggestion
	var m = new DOMMatrixReadOnly(transform);
```